frontend/packages/text-editor/src/Variables.module.css

```diff
@@ -5,15 +5,34 @@
 
 .chip {
   border-radius: 20px;
-  width: 90%;
   margin: 5px;
   background-color: gray;
+  padding-left: 10px;
+  vertical-align: text-bottom;
+}
+
+.keyDataSource {
   display: inline-block;
   overflow: hidden;
+  width: 90%;
+  white-space: nowrap;
   text-overflow: ellipsis;
+  color: white;
+  font-size: 14px;
+}
+
+.defaultValue {
+  display: inline-block;
+  overflow: hidden;
+  width: 90%;
   white-space: nowrap;
+  text-overflow: ellipsis;
   color: white;
-  padding-left: 10px;
   font-size: 14px;
-  vertical-align: text-bottom;
 }
```

Comment:
Nothing really important here, but `.keyDataSource` and `.defaultValue` seem to have the exact same style. They could probably be merge like this :

```
.keyDataSource,
.defaultValue {
  display: inline-block;
  overflow: hidden;
  width: 90%;
  white-space: nowrap;
  text-overflow: ellipsis;
  color: white;
  font-size: 14px;
}
```